### PR TITLE
Re-invite author to team

### LINKED ISSUE #67: Command to re-issue an invitation to the org and team
Currently the approve command issues an invitation to the package author.

That invitation expires after one week.

Could there be a command that the author uses to re-issue an invitation, maybe `@ropensci-review-bot re-invite me to ropensci`?

Better ideas for words @mpadge?

### PR DESCRIPTION
**Pull request by xuanxu**
Currently the approve command issues an invitation to the package author but that invitation expires after one week.

This PR adds a responder for a new command `invite me to ropensci/team` that the author of a recently transferred package can use to receive again the invitation to the team managing the package at rOpenSci.

Closes #67

### CODE REVIEW

File: docs/responders/ropensci/invite_author.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Listens to
+
+```
+@botname invite me to ropensci/package-name
+```
+
+## Requirements
+
+The _ropensci/package-name_ team must exist and the _package-name_ must be a package already transfered to rOpenSci, otherwise an error message will be sent as reply.
```
Comment (maelle) [inline docs/responders/ropensci/invite_author.md:14]: At the point the author receives the new invitation to the team and org, they cannot have already transferred the package as they need to be organization member for that?

**Review by maelle — COMMENTED** [review]
Thank you! I have a question, see below.

File: docs/responders/ropensci/invite_author.md (hunk 1 above)
Comment (maelle) [reply docs/responders/ropensci/invite_author.md:14]: yes indeed!
thank you!

Comment (xuanxu) [reply docs/responders/ropensci/invite_author.md]: Done!

**Comment by maelle** [comment]
Thank you!

**Pull request merged**